setenv.sh

```diff
@@ -0,0 +1,19 @@
+# BITGO configuration
```

Comment:
You're right. I only did this because I didn't want config changes to force a rebuild of the Docker image. This is no longer an issue (the `config` folder is mounted to the container at runtime, after being copied to the relevant Docker host machine if you're using docker-machine) and I have deleted `setenv.sh`. Thanks!